Add Course Comparator Project

108anup opened the pull request.

swapagarwal commented on Pull Request:
```diff
@@ -0,0 +1,6 @@
+I don't know how to make pull request from a repository that has not been forked from the hackathon repository.
+Please guide me.
```
You've already forked it.
Now edit the README file and replace the "Sample Project" placeholder with your project's details.

108anup commented:
I have edited the Readme.md file

swapagarwal commented:
Cool, but there are 3 commits and I'd prefer squashing (google is your friend) this into a single commit. Let me know if you need any help.

108anup commented:
Done, Confirm the changes

swapagarwal commented:
Great!
I just noticed you changed the date as well. That was the hackathon date!

108anup committed:
Add Course Comparator

Add Course Comparator

Allows Students of Different Universities to Compare Courses offered at other Universities.

108anup commented:
ok changed the date, is it fine now?

swapagarwal commented:
Nice work @108anup :+1:
